tests/fixtures.py

```diff
@@ -28,6 +31,32 @@ def load_config():
         raise Exception("KUBECONFIG file not found: {0}".format(kube_config_path))
 
 
+@pytest.fixture(scope='session')
+def set_namespace(request, load_config):
+
+    def _setup_namespace():
+        k8s_namespace = os.getenv('K8S_NAMESPACE', '')
+        if k8s_namespace != '':
+            testconfig['namespace'] = k8s_namespace
+
+        print("Run tests in namespace: {0}".format(testconfig['namespace']))
+        v1 = client.CoreV1Api()
```

Comment:
if you declare `v1` under `set_namespace` you could use it both `_setup_namespace` and `fin()`. right ?

Reply:
right. I'll fix it